createwiki.sh

```diff
@@ -45,6 +45,14 @@ php $PATCHDEMO/wikis/$NAME/w/maintenance/install.php \
 --pass=patchdemo1 \
 "$WIKINAME" "Patch Demo"
 
+# create additional accounts
+# generic accounts alice/bob e.g. for messaging tests
+php $PATCHDEMO/wikis/$NAME/w/maintenance/createAndPromote.php Alice patchdemo1
+php $PATCHDEMO/wikis/$NAME/w/maintenance/createAndPromote.php Bob patchdemo1
+# blocked account
+php $PATCHDEMO/wikis/$NAME/w/maintenance/createAndPromote.php Mallory patchdemo1
+echo "Mallory" | php $PATCHDEMO/wikis/$NAME/w/maintenance/blockUsers.php --performer "Patch Demo" --reason "Blocking account for testing"
```

Comment:
-1 This creates an autoblock for the IP (which everyone uses) if Mallory tries to edit